### PR TITLE
Allow bootstrap directory and nuget packages to be configured

### PR DESCRIPTION
**Pull request by karajas**
- If this PR should not run tests please add text "skip[REMOVE_THIS]ci[REMOVE_THIS]please" (remove the marked text, no quotes).
- Please add description for changes you are making.
- If there is an issue related to this PR, please add the reference.

This change allows configuration of the download of the bootstrap.ps1 script and other tools.

### CODE REVIEW

**Comment by piotrpMSFT** [comment]
Why is this change not needed in the .sh file?

**Comment by karajas** [comment]
@piotrpMSFT Sorry didn't stage it.

**Comment by eerhardt** [comment]
Please remove the skip ci please. You are changing the build scripts, so we require CI to be green.

**Comment by karajas** [comment]
@dotnetbot test this please

**Comment by karajas** [comment]
@dotnet-bot test Windows_NT x86 Debug Build please

File: run-build.ps1 [hunk 1]
```diff
@@ -29,7 +29,9 @@ if($Help)
 
 $env:CONFIGURATION = $Configuration;
 $RepoRoot = "$PSScriptRoot"
+if(!$env:NUGET_PACKAGES){
 $env:NUGET_PACKAGES = "$RepoRoot\.nuget\packages"
```
Comment (eerhardt) [inline run-build.ps1]: (nit) should be tabbed since it is now in a if block.

**Pull request merged**